saml/saml.php

```diff
@@ -119,10 +120,7 @@ function saml_sso_initiate(App $a, array &$b)
 	$auth = new \OneLogin\Saml2\Auth(saml_settings());
 	$ssoBuiltUrl = $auth->login(null, [], false, false, true);
 	$_SESSION['AuthNRequestID'] = $auth->getLastRequestID();
-	header('Pragma: no-cache');
-	header('Cache-Control: no-cache, must-revalidate');
```

Comment:
Please keep the cache control headers, I’m not sure what would be the side-effects of removing them.

Reply:
Or cannot we move them to the method `System::externalRedirect()`?

Reply:
Moving the headers to the function could have got other side effects.

Reply:
Okay, I would like to have this easier for addon developers, like they don't need to care so much by themselves about it. I will add another parameter to the method.

Reply:
`require_once(__DIR__ . '/vendor/autoload.php');` <-- Can this line be removed?